client/app/dashboard/setup/bookshelf/page.tsx

```diff
@@ -2,17 +2,19 @@
 
 import { AccordionManager, Checkbox } from "@/components";
 import { IncludeDefaultShelves, SetupBookshelf, useSetupWizard } from "@/hooks";
+import { useUser } from "@auth0/nextjs-auth0/client";
```

Comment:
Reckon should move this call to the setup modal instead -> less duplication. 

Will be completed in a follow up PR